.github/actions/build-prqlc/action.yaml

```diff
@@ -0,0 +1,54 @@
+name: build-prqlc
+description: >
+  Build prqlc
+inputs:
+  target:
+    description: Build target
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - run: rustup target add ${{ inputs.target }}
+      shell: bash
+
+    - uses: Swatinem/rust-cache@v2
+      with:
+        save-if: ${{ github.ref == 'refs/heads/main' }}
+        prefix-key: 0.8.1
+
+    - if: runner.os == 'Linux'
+      shell: bash
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y musl-tools
+
+    - if: inputs.target == 'aarch64-unknown-linux-musl'
+      shell: bash
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y gcc-aarch64-linux-gnu
+        echo 'CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_LINKER=aarch64-linux-gnu-gcc' >>"$GITHUB_ENV"
+        echo 'RUSTFLAGS=-C link-arg=-lgcc' >>"$GITHUB_ENV"
```

Comment:
I think this is the correct way to specify it, since enclosing it in quotation marks failed.
Searching across GitHub, I can see other projects that do it this way.